models/src/main/java/com/connexta/transformation/rest/models/TransformationPollResponse.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/** TransformationPollResponse */
+@javax.annotation.Generated(
+    value = "org.openapitools.codegen.languages.SpringCodegen",
+    date = "2019-11-04T14:07:00.984818900-07:00[America/Phoenix]")
+public class TransformationPollResponse {
+  @JsonProperty("transformationStatus")
+  private Status transformationStatus;
+
+  @JsonProperty("errorMessage")
+  private ErrorMessage errorMessage = null;
+
+  @JsonProperty("metadataInformations")
+  @Valid
+  private List<MetadataInformation> metadataInformations = new ArrayList<>();
+
+  public TransformationPollResponse transformationStatus(Status transformationStatus) {
+    this.transformationStatus = transformationStatus;
+    return this;
+  }
+
+  /**
+   * Get transformationStatus
+   *
+   * @return transformationStatus
+   */
+  @ApiModelProperty(required = true, value = "")
+  @NotNull
+  @Valid
+  public Status getTransformationStatus() {
+    return transformationStatus;
+  }
+
+  public void setTransformationStatus(Status transformationStatus) {
+    this.transformationStatus = transformationStatus;
+  }
+
+  public TransformationPollResponse errorMessage(ErrorMessage errorMessage) {
```

Comment:
❗️ This should not be the `ErrorMessage` used as a response body for errors. It should only contain the failure code and message string.